tools/reflection_generator/reflection_generator.py

```diff
@@ -127,6 +129,8 @@ def main(argv):
   option_parser.add_option('--target-arch', help='Target Architecture')
   option_parser.add_option('--sdk-version', help='API Version')
   option_parser.add_option('--min-sdk-version', help='Min API Version')
+  option_parser.add_option('--verify-xwalk-apk',
```

Comment:
I think it makes sense to pass `type="int"` here to make sure you get consistent values.